addons/sys_intercom/fnc_getStationVariableName.sqf

```diff
@@ -0,0 +1,44 @@
+/*
+ * Author: ACRE2Team
+ * Gets the variable name for accessing intercom configuration in the players position.
+ *
+ * Arguments:
+ * 0: Vehicle with intercom <OBJECT>
+ * 1: Unit to be checked <OBJECT>
+ *
+ * Return Value:
+ * Variable name, string of length 0 if not found <STRING>
+ *
+ * Example:
+ * [vehicle acre_player, acre_player] call acre_sys_intercom_fnc_getStationVariableName
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_vehicle", "_unit"];
+
+private _found = false;
+private _varName = QGVAR(station_);
+{
+    if (_unit isEqualTo (_x select 0)) then {
+        private _role = toLower (_x select 1);
+        _varName = _varName + _role;
+        if (_role in ["cargo", "turret"]) then {
+            if (_role isEqualTo "cargo") then {
+                _varName = _varName + format ["_%1", _x select 2];
```

Comment:
`_varName = format ["%1_%2", _varName, _x select 2];`, `+` is a lot slower. Same below.